DockerIntegrationTest fails when boot2docker is not present

```
[INFO] Running org.wildfly.camel.test.docker.DockerIntegrationTest
Nov 13, 2017 9:20:36 AM org.arquillian.spacelift.Spacelift$SpaceliftInstance <init>
INFO: Initialized Spacelift from defaults, workspace: /Users/tdiesler/git/wildfly-camel/itests/standalone/docker, cache: /Users/tdiesler/.spacelift/cache
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.723 s <<< FAILURE! - in org.wildfly.camel.test.docker.DockerIntegrationTest
[ERROR] org.wildfly.camel.test.docker.DockerIntegrationTest  Time elapsed: 2.722 s  <<< ERROR!
java.lang.RuntimeException: Could not create new instance of class org.jboss.arquillian.test.impl.EventTestRunnerAdaptor
Caused by: java.lang.reflect.InvocationTargetException
Caused by: org.arquillian.spacelift.execution.ExecutionException: Execution of a task failed. Execution of a task failed. Unable to execute task SpawnProcessTask
Caused by: java.io.IOException: Cannot run program "boot2docker": error=2, No such file or directory
Caused by: java.io.IOException: error=2, No such file or directory
```

--- a/itests/standalone/docker/src/test/java/org/wildfly/camel/test/docker/DockerIntegrationTest.java
+++ b/itests/standalone/docker/src/test/java/org/wildfly/camel/test/docker/DockerIntegrationTest.java
@@ -26,8 +26,8 @@ import org.apache.camel.ProducerTemplate;
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.impl.DefaultCamelContext;
 import org.arquillian.cube.docker.impl.requirement.RequiresDocker;
+import org.arquillian.cube.requirement.ArquillianConditionalRunner;
 import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.Assert;
@@ -38,7 +38,7 @@ import org.wildfly.camel.test.common.utils.TestUtils;
 import org.wildfly.extension.camel.CamelAware;
 
 @CamelAware
-@RunWith(Arquillian.class)
+@RunWith(ArquillianConditionalRunner.class)
 @RequiresDocker
 public class DockerIntegrationTest {
 

